chingu-voyages/v51-tier3-team-33
# removed the package-lock.json file

# Description
Deleted the `package-lock.json` file and left the `pnpm-lock.yaml` file as the config for deployment use.

## Related to the following Jira/Trello issue 
Not related to a user story...

## Main changes explained:
- Delete package-lock.json

## How to test:
1. check into current branch
2. do `pnpm install` and `npm run dev` to run this PR locally
3. assert that the project runs correctly.

## Screenshots or videos of changes:
![chat gpt response to npm or pnpm](https://github.com/user-attachments/assets/d0486bca-198d-4f9e-9d98-bbe0ba9d84dd)

## Note:
Have fun reviewing the changes
